dataflux_pytorch/benchmark/README.md

```diff
@@ -14,7 +14,7 @@ pip install gcs-torch-dataflux gcsfs
 
 First ensure you are running within a virtual python enviroment, then set the enviroment variables.
 
-`CKPT_DIR_PATH` is the location of where to save the checkpoints. `STEPS` is the number of steps the model will take (the number of checkpoints created will be the same). The default value for `STEPS` is 5.
+`CKPT_DIR_PATH` is the location of where to save the checkpoints. `STEPS` is the number of steps the model will take (the number of checkpoints created will be the same). The default value for `STEPS` is 5. The benchmark will run `save_checkpoint` repeatively and produce the average at the end.
 
 ```shell
 export CKPT_DIR_PATH=`gs://path/to/directory/`
```

Comment:
I think if possible we should step away from modify the global variables and instead, use argparser here so we can add those flags directly to the Python program. There are other demos in dataflux-pytorch for doing that.

Macros are tough to manage as the variables are not obvious as the flags.

Reply:
Not now but in later PRs.